includes/platform-checkout-user/class-platform-checkout-save-user.php

```diff
@@ -57,11 +58,17 @@ public function register_checkout_page_scripts() {
 	 * @return array
 	 */
 	public function maybe_add_userdata_to_metadata( $metadata, $order ) {
+		$session_data = WC()->session->get( 'platform-checkout-user-data' );
```

Comment:
```suggestion
		$session_data = WC()->session->get( Platform_Checkout_Extension::PLATMORM_CHECKOUT_SESSION_KEY );
```
Sorry, you're going to see me suggesting this a lot now that I've noticed it...but you could use your constant here!